pkg/engine/cmd.go

```diff
@@ -20,6 +20,11 @@ import (
 	"github.com/gptscript-ai/gptscript/pkg/version"
 )
 
+var requiredFileExtensions = map[string]string{
+	"powershell.exe": "*.ps1",
+	"powershell":     "*.ps1",
```

Comment:
Why are there asterisks on these? I'm a bit confused

Reply:
The `*` allows Go to add randomness in the temporary file name.